base/server/share/lib/systemd/system/pki-tomcatd@.service

```diff
@@ -9,7 +9,7 @@ EnvironmentFile=/etc/tomcat/tomcat.conf
 Environment="NAME=%i"
 EnvironmentFile=-/etc/sysconfig/%i
 
-ExecStartPre=/usr/sbin/pki-server upgrade --validate %i
```

Comment:
Before this patch, the upgrade script was executed when `dnf update` was executed and then validated when the service was restarted. 

With this patch, we are running only the upgrade and we are not validating it. Do we want to include an additional `ExecStartPre` after we upgrade just to validate the upgrade process?

Reply:
Here if the upgrade process fails the systemd service will not start, so essentially it's also validating the upgrade process. If that happens, the admin will need to investigate and fix problem manually, or bypass the upgrade, which is the same as before.